src/relations/ui.py

```diff
@@ -42,7 +42,7 @@ def _on_ui_relation_joined(self, event):
     def _provide_server_status(self):
         """Provide server status to the UI charm."""
         charm = self.charm
-        is_active = charm.model.unit.status == ActiveStatus()
+        is_active = charm.model.unit.status == ActiveStatus() or charm.model.unit.status == ActiveStatus("auth enabled")
```

Comment:
What about using Pebble to fetch the status of one of your containers checks here instead? https://juju.is/docs/sdk/interact-with-pebble#heading--fetch-check-status

Reply:
I've added this as a todo as the checks I tried enabling were not functioning properly.